test/controllers/species_list_controller_test.rb

```diff
@@ -1116,6 +1116,26 @@ def test_make_report
     end
   end
 
+  def test_download
+    spl = species_lists(:one_genus_three_species_list)
+    query = Query.lookup_and_save(:Observation, :in_species_list,
+                                  species_list: spl)
+
+    get_with_dump(:download, id: spl.id)
```

Comment:
```suggestion
 get(:download, id: spl.id)
```
My memory is that it was okay to get rid of the `with_dump` methods (because we were't checking the validity of the generated html)